src/components/DelegationNode/DelegationNode.tsx

```diff
@@ -276,6 +276,13 @@ class DelegationNode extends React.Component<Props, State> {
 
     const hashes = await delegation.getAttestationHashes()
 
+    if (hashes.length < 1) {
```

Comment:
Lets maybe not include these changes in a temp fix PR, after we talk tomorrow, these changes might need to be reverted again.

Reply:
That's true, but I doubt we'd do a revert of the PR. More likely we'd change all occurrences of `BlockchainUtils.submitTxWithReSign` to whatever we come up in the redesign.
I'd say if restructured some of the code to be a little cleaner here, why would we want to throw that away later.

Reply:
but I can of course pull it in a separate PR, doesn't hurt